### ISSUE TITLE
请问解码的在解码的时候，ememory_vocab是否用到了？

### ISSUE THREAD

**Issue by bearblog** [tuxchow/ecm#20]
else:

                        # This is a greedy decoder - outputs are just argmaxes of output_logits.
                        outputs = [int(np.argmax(np.split(logit, [2, FLAGS.response_vocab_size], axis=1)[1], axis=1)+2) for logit in output_logits]
                        # If there is an EOS symbol in outputs, cut them at that point.
                        if data_utils.EOS_ID in outputs:
                            outputs = outputs[:outputs.index(data_utils.EOS_ID)]
                        # Print out response sentence corresponding to outputs.
                        print(int2emotion[decoder_emotion]+': '+"".join([tf.compat.as_str(rev_response_vocab[output]) for output in outputs]))
                print("> ", end="")

在baseline.py这个文件中解码时没有体现使用外部词表， @tuxchow 请问这个是为什么呢？
